tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 project = pre_commit
 # These should match the travis env list
-envlist = py27,py35,py36,pypy
+envlist = py27,py35,py36,py37,pypy
```

Comment:
when adding py37, we can probably remove py35 -- we don't have any python3.5 specific code and I want to keep travis-ci fast :)